### DISCUSSION TITLE
sqlitestudio 3.2.1 portable -  version `Qt_5.10' not found (required by ./sqlitestudio)

### DISCUSSION THREAD

**Discussion by mbezzi** [pawelsalawa/sqlitestudio#3408] [Q&A]
Hello, 
I just tried to run the portable version of sqlitestudio 3.2.1 in my Ubuntu 18.04, and I get the following message:

./sqlitestudio: /home/mysef/bin/SQLiteStudio/lib/libQt5Core.so.5: version `Qt_5.10' not found (required by ./sqlitestudio)

Looks like sqlitestudiopl 3.2.1 is compiled against a vary recent version of QT. Ubuntu 18.04 just released just 3 monts ago, and I hate the idea of manually installing another version of QT. 

I imagine other users are experiencing the same? Or is it me?

Would it be possible to have a new package compiled using a more commonly available version of QT?

Thank you,
mario

**Comment by pawelsalawa** [comment]
The Qt in proper version is shipped together with SQLiteStudio in the same package. It should be using this one.

Can you please tell me results of these 2 commands (while being in directory `/home/mysef/bin/SQLiteStudio`):

`ldd sqlitestudio`

and then also:

`ldd lib/libQt5Core.so.5`

**Comment by RazbojNick** [comment]
I had to rename all the libs in the `lib` folder by removing from the name everything from the version part but the first number, eg. `libQt5Core.so.5.10.0` became `libQt5Core.so.5, `. Aftrer that the portable version worked. Ubuntu 18.04.

**Comment by pawelsalawa** [comment]
Still, it should not even need to do this... Did you use binary package from sqlitestudio.pl?

**Comment by RazbojNick** [comment]
Yes, Linux files (64bit) sqlitestudio-3.2.1.tar.xz (portable) from Releases page.

In addition, the `ldd sqlitestudio` command confirms the issue.

**Comment by pawelsalawa** [comment]
Can you please also tell me what this command shows you: `ls -l lib | grep Core` ?

**Comment by RazbojNick** [comment]
`ls -l lib | grep Core`:

> -rwxrwxrwx 1 sus sus  5544440 jul 27  2018 libQt5Core.so.5

**Comment by pawelsalawa** [comment]
That's the problem. There should be symlinks, like this:
![image](https://user-images.githubusercontent.com/481846/82425589-01390d00-9a87-11ea-90fd-a0c18bd085b8.png)

What did you use to decompress the tar.xz package?

**Comment by RazbojNick** [comment]
Ubuntu's built-in Archive manager, just unpacked the archive contents to my /home folder

**Comment by pawelsalawa** [comment]
Do you mean like this one?
![image](https://user-images.githubusercontent.com/481846/82427217-11ea8280-9a89-11ea-8c39-c5d5e6030f08.png)

I've tried that and it works correctly. I'm not sure what went wrong for you, but definitely you ended up with incorrectly decompressed package.

**Comment by RazbojNick** [comment]
That is weird. I just did that again and there are symlinks in lib, but the original one doesn't have them. I haven't even gone into the libs folder until I read this issue here... All I got from running `sqlitestudio` in terminal is 

> sqlitestudio: command not found
